library/packages/src/modules/PackagesUI.rb

```diff
@@ -312,7 +312,7 @@ def RunPackageSelector(options)
         widget_options
       )
 
-      UI.OpenDialog(
+      raise "Opening package selector failed." if !UI.OpenDialog(
```

Comment:
Should we translate the error message? We do so for the other errors so it would make sense also here IMHO.

Reply:
I could not find other exceptions being translated so I also did not translate this one. But surely I can add translations.

Reply:
BTW: Most of the messages in the generic exception popup itself are not translated.

Reply:
in general we should not translate text in exceptions as generic exceptions catcher should be as trivial as possible as even I18n can be down or in inconsistent state. Where translation makes sense is place where you catch such specific exception and where you provide some nice message for user what he can do.